index.css

```diff
@@ -847,34 +848,119 @@ button :disabled {
   background: #f4f1f0;
 }
 
-.other_works_item :hover + .action {
+.required::before {
+  content: "*";
+  color: red;
+  float: right;
+}
+
+/* The Modal (background) */
+#myModal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+/* Modal Content */
+.modal_content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%;
+}
```

Comment:
[MOBILE] Please make sure to adjust the size of your technologies on the mobile popup 😉: 
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/209198547-e2bf8287-70a1-4c0b-8ee6-206b646f385f.png)  | ![image](https://user-images.githubusercontent.com/42799579/209198516-ceb5c037-a042-4dd5-8dda-f511094d3d58.png)  |

Check that, on your design, the height of these technologies is higher than Figma's.

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.